### ISSUE TITLE
Importing .fbx files including animations from 3DsMax ( with dirty solution )

### ISSUE THREAD

**Issue by Consti10** [google-ar/sceneform-android-sdk#648]
Hello,
Since animations are mandatory for our AR app we have been experimenting with sceneform for quite a while, but until now we were unable to succesfully import files created by 3DsMax in Sceneform with Animations.
Our testing included
a) using different assets from our artists
b) making sure the animations work in Unity and Unreal
c) making sure to apply all settings listed here https://developers.google.com/ar/develop/java/sceneform/animation/obtaining-assets
But no success with sceneform.

However, working with the andy_dance as resource with different programs I figured out that exporting with 3DsMax to Sceneform must be missing some option / default param,
because when **importing and then exporting the andy_dance in 3DsMax the animations are gone in Sceneform** (gone means 'not playable by the viewer or in java' but the data is there)

However, Importing the file from 3DsMax in Blender  afterwards made its **animation reappear** in Sceneform
( andy_dance.fbx -> import in 3ds max -> export from 3ds max -> | unplayable in Sceneform | -> import in blender -> export from blender -> | suddenly playable again in Sceneform | )

Also, I confirmed that files created (and exported) by our artists in 3DsMax can be fixed by *passing them through blender*

While this is a 'solution' for everyone having problems with 3dsmax and sceneform, I would appreciate if you could look into that.
Perhaps there is some mandatory setting not enabled when exporting from 3DsMax, but enabled by default when exporting from blender.

Thanks !

**Comment by Consti10** [comment]
Since the andy_dance model was created by Autodesk maya, I also used it instead of blender in the 'reimport to fix animations' flow and the result is the same as with blender:
Using Autodesk Maya, Importing & exporting a .fbx file created by 3DsMax with animations makes them reapperar in sceneform.

**So I can clarify the Issue here:**
Exporting animated models with 3DsMax to Sceneform with .fbx as format does not work.
The animation data must be 'still inside' the exported .fbx file, but is not recognizable by sceneform.
Only when importing/exporting the model again using either Autodesk Maya or blender they are recognized by sceneform.

**Comment by Consti10** [comment]
Hello,
We did some more testing and found a better solution.
1) Background: Someone in our team remembered that when importing from A3DsMax to unreal, it can be neccesary to export a)rig model and b) bones with animations seperately, e.g.
biene_model.fbx and
biene_animation.fbx.
I assume it is done in a similar way with andy_wave_left and right.
While doing exactly as described did not work exactly due to the following error message when viewing it in android studio:
`java.lang.RuntimeException: Model(target) expects to resample 54 bones; Animation(source) has 51 bones
	at com.google.ar.sceneform.assets.AnimationInstance.nCreateAnimationTarget(Native Method)
	at com.google.ar.sceneform.assets.AnimationInstance.createTarget(AnimationInstance.java:23)
	at com.google.ar.sceneform.plugin.viewer.FilamentViewer.setupAnimation(FilamentViewer.java:124)
	at com.google.ar.sceneform.plugin.viewer.FilamentViewer.access$500(FilamentViewer.java:32)
	at com.google.ar.sceneform.plugin.viewer.FilamentViewer$1$1.run(FilamentViewer.java:97)
	at com.google.ar.sceneform.plugin.viewer.Filamentor$3.run(Filamentor.java:113)`

We managed to bypass this error by adding the animation to biene_model.fbx and then importing biene_model.fbx only, but selecting the file 2 times in the Sceneform importer.
1) as Source assets path
2) as Animation files.

This way we finally have animations directly when importing from Autodesk, even though testing took us quite some time and we are still left with #650 :

No textures for our animated models ( textures only work with .obj files out of the box)
